include/cute/arch/mma_xe.hpp

```diff
@@ -34,9 +34,8 @@
 #define CUTE_ARCH_MMA_XE_ENABLED
 #endif
 
-#if defined(CUTE_ARCH_MMA_XE_ENABLED) && ((defined(__INTEL_LLVM_COMPILER) && (__INTEL_LLVM_COMPILER < 20250200)) || defined(CUTLASS_SYCL_BUILTIN_ENABLE))
-#include<cute/arch/mma_xe_builtin.hpp>
```

Comment:
I guess you should also delete the file `mma_xe_builtin.hpp`?

Reply:
done